pkg/k8s_imported/cpuset.go

```diff
@@ -4,6 +4,13 @@ Copyright 2017 The Kubernetes Authors.
 This was copied out of kubernetes source to avoid pulling too
 many dependencies with the full kubernetes source code.
 
+This is based on the following kubernetes file:
+
+https://github.com/kubernetes/kubernetes/blob/release-1.26/pkg/kubelet/cm/cpuset/cpuset.go
```

Comment:
please comment (GH or better here) which sha256 was used (`release-1.26` will always point to the tip of that branch)

Reply:
Good point.